libs/srt/Makefile

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2024 Koen Vandeputte <koen.vandeputte@citymesh.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=srt
+PKG_VERSION:=1.5.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/haivision/srt/tar.gz/v$(PKG_VERSION)?
```

Comment:
PKG_SOURCE_PROTO:=git

Reply:
Switching to Git? No need, I would say when upstream is tagging releases.

Reply:
I would say local tarballs are preferable to code load.

Reply:
Dozens and dozens of other packages use the exact same way:
`grep -r "PKG_SOURCE_URL" | grep codeload`

Reply:
Sure. We probably want to move away from them.

Reply:
Not sure, because if the verifying checking out sources fails, then it downloads a new one and tries to compile it, which is IMHO wrong behavior. This was discussed on the mailing list a few months ago with the xz drama. If you are downloading tarball and the checksum does not match, then the package fails entirely.

Reply:
Details here: https://lists.openwrt.org/pipermail/openwrt-devel/2024-April/042521.html

So, we should ping @ynezz because, if I am not mistaken, there isn't anything new, and we need a statement on what we should use and keep using it looks like last year, we preferred codeload, but now you want to use Git checkout. :-/ Is there any issue in the main repo to look at it?